krun/scheduler.py

```diff
@@ -189,6 +182,15 @@ def __init__(self, sched, vm_name, vm_info, benchmark, variant, parameter):
         # Used in results JSON and ETA dict
         self.key = "%s:%s:%s" % (self.benchmark, self.vm_name, self.variant)
 
+        # When a benchmark fails, you get this:
+        per_core = [[]] * sched.platform.num_per_core_measurements
```

Comment:
This isn't safe:

```python
>>> a = [[]] * 2
>>> a[0].append(10)
>>> a
[[10], [10]]
```
Even if, in this case, the sub-lists aren't used, the idiom is a classic shoot-in-the-footer. Best to do something like:
```python
per_core = []
for _ in range(sched.platform.num_per_core_measurements: per_core.append([])
```